### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by rbd3**
### Change made:
👉 Add Style for all components

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,4 +1,283 @@
...
+  --max-width: 1000px;
+}
+
+.nav-container {
+  display: grid;
+  place-items: center;
+  border-bottom: 2px solid var(--border-color);
+  margin-bottom: 2%;
+  margin-left: -5rem;
+}
```
Comment (VICTOR-CHUKWU) [inline src/App.css]: - [ ] This is a style ,milestone so we have to make sure the styling is close to the given design, currently your nav is not aligned to the body of the booklist, kindly make it so
your design :point_down: 
image
given design :point_down: 
image

File: src/App.css [hunk 2]
```diff
@@ -1,4 +1,283 @@
...
+.link-container a {
+  text-decoration: none;
+  font-size: 20px;
+  letter-spacing: 2px;
+  color: var(--text-color);
+}
+
+.link-container a.active {
+  font-weight: 700;
+}
```
Comment (VICTOR-CHUKWU) [inline src/App.css]: - [ ] the link texts are not looking like what is on design, kindly fix the font color

File: src/App.css [hunk 3]
```diff
@@ -1,4 +1,283 @@
...
+
+.form-container {
+  display: grid;
+  margin-left: 8rem;
+}
+
+.form-wrapper {
+  max-width: var(--max-width);
+  width: 80%;
+}
```
Comment (VICTOR-CHUKWU) [inline src/App.css]: - [ ] kindly adjust the width of this wrapper to look like the given design
your design :point_down: 
image
given design :point_down: 
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @rbd3  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 4]
```diff
@@ -1,4 +1,283 @@
...
+  font-size: 1.3rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-title {
```
Comment (tufoinnkuo10) [inline src/App.css:143]: - [ ] wonderful job so far. please, like the previous reviewer  noted, this milestone is aimed at our styling looking almost like the zeplin design. Kindly align the book cards with the page title and user logo placeholder like in the original design. 
yours
Screenshot (1267)_LI
original
Screenshot (1266)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @rbd3 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ wireframe well followed

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @rbd3 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**